src/client/api/json_parser/inputs/inputs.c

```diff
@@ -54,7 +54,7 @@ int json_inputs_deserialize(cJSON *inputs_obj, utxo_inputs_list_t **inputs) {
 }
 
 cJSON *json_inputs_serialize(utxo_inputs_list_t *inputs) {
-  char tx_id_str[BIN_TO_HEX_STR_BYTES(IOTA_TRANSACTION_ID_BYTES)] = {};
+  char tx_id_str[BIN_TO_HEX_STR_BYTES(IOTA_TRANSACTION_ID_BYTES) + JSON_HEX_ENCODED_STRING_PREFIX_LEN] = {};
```

Comment:
it's long declaration, should define another macros for hex with prefix?
```suggestion
  char tx_id_str[BIN_TO_HEX_STR_WITH_PREFIX_BYTES(IOTA_TRANSACTION_ID_BYTES)] = {};
```

Reply:
Additional macro was introduced `JSON_STR_WITH_PREFIX_BYTES(x)`.